### ISSUE TITLE
Undocumented breaking change from 0.4 -> 0.7 importing global JS objects

### ISSUE THREAD

**Issue by tomcl** [fable-compiler/Fable#623]
### Description

Importing global objects from npm JS modules used to be implemented irregularly as:

```
type Globals =[<Global>] static member CodeMirror with get(): CodeMirrorType = failwith "JS only" and set(v: CodeMirrorType): unit = failwith "JS only"
```
with the module JS explicitly referenced in the html file.

### Expected behavior

Previously this statement caused `Globals.CodeMirror` to refer correctly to the global `CodeMirror` object.

### Actual behavior

This no longer works. After this declaration (with `CodeMirror` correctly globally in scope) references to `Globals.CodeMirror` in F# transpile to `get_CodeMirror` which however is not defined.

### Known workarounds

The new (much better) way to do this is documented an in issue [here](https://github.com/fable-compiler/Fable/issues/587).

### Related information 

It is worth highlighting this breaking change because the old documentation specifies (rather vaguely) the old method, without a clear alternate, the transition to 0.7 chnage notes don't deal with the issue, and there is no obvious sample to follow so correcting this takes a while.

* Fable Version: 0.7.17
4.6 .NET Runtime
W10 Operating system

**Comment by alfonsogarciacaro** [comment]
Hmm, this doesn't look like a breaking change but a regression bug, sorry! 😅 I'll fix it.

**Comment by alfonsogarciacaro** [comment]
This should be fixed in fable-compiler 0.7.25 :+1:

**Comment by tomcl** [comment]
Thanks!
